packages/cdk8s/README.md

```diff
@@ -85,7 +217,6 @@ cdk8s bundles a set of test utilities under the `Testing` class:
   chart.
 
 
```

Comment:
Can you also add documentation on how to use the new `Synthesis` methods?

Reply:
Is this relevant? We haven't added anything to testing capabilities, `Testing.synthChart` still calls `chart.toJson()`...